src/nvim/os/shell.c

```diff
@@ -68,6 +68,18 @@ char **shell_build_argv(const char *cmd, const char *extra_args)
   return rv;
 }
 
+char **build_argv(char *cmd)
```

Comment:
This function is nearly identical to `shell_build_argv`, why not use `shell_build_argv(cmd, NULL)` instead?